Doctrine/Enum/PluginType.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AdyenPayment\Doctrine\Enum;
```

Comment:
this is not a Doctrine Enum type, it's created for our Plugin.
Refactor to a namespace outside Doctrine.

Reply:
Is this class actually used?